cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureScoringTypeContinuousVariableTest.java

```diff
@@ -108,28 +107,15 @@ void continuousVariableResourcePopulation() {
 
     @Test
     void continuousVariableBooleanMissingRequiredPopulation() {
-        try {
-            given.when()
-                    .measureId("ContinuousVariableBooleanMissingReqdPopulation")
-                    .evaluate()
-                    .then()
-                    .firstGroup()
-                    .population("initial-population")
-                    .hasCount(10)
-                    .up()
-                    .population("measure-population")
-                    .hasCount(10)
-                    .up()
-                    .population("measure-population-exclusion")
-                    .hasCount(10)
-                    .up()
-                    .up()
-                    .report();
-            fail("This should throw error");
-        } catch (UnsupportedOperationException e) {
-            assertTrue(e.getMessage()
-                    .contains("'continuous-variable' measure is missing required population: initial-population"));
-        }
+        given.when()
+                .measureId("ContinuousVariableBooleanMissingReqdPopulation")
+                .evaluate()
+                .then()
```

Comment:
Question:  are you expecting this operation to have a status of ERROR like the other tests?